curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/iterate-through-the-keys-of-an-object-with-a-for...in-statement.md

```diff
@@ -15,10 +15,10 @@ for (let user in users) {
 }
 
 // logs:
-Alan
-Jeff
-Sarah
-Ryan
+Alan;
+Jeff;
+Sarah;
+Ryan;
```

Comment:
@ojeytonwilliams I assume this has to do with the new syntax (confusing the prettier config). Can we avoid this?

Reply:
I think i fixed this

Reply:
> @ojeytonwilliams I assume this has to do with the new syntax (confusing the prettier config). Can we avoid this?

This isn't new, it's just how prettier handles fenced code blocks.  

We can mitigate this, but it involves identifying specific cases (like this) where prettier will break something.  Nick's taking a look at this, so we can discuss a bit further when he drops his PR.